.github/workflows/main.yml

```diff
@@ -1,18 +1,21 @@
-name: main
+name: Install and Lint
```

Comment:
Can this file go away? No need to build and lint on push if we have the PR build in place and we aren't doing a deploy.

Reply:
NM, I'm tracking with you.  I've removed the former workflow file.